Modelica/Electrical/Polyphase/Examples/Utilities/PolyphaseRectifierData.mo

```diff
@@ -12,11 +12,14 @@ record PolyphaseRectifierData "Data record for polyphase rectifier"
     annotation(Dialog(enable=false));
   final parameter Integer kPolygon=max(1, integer((mBasic - 1)/2))
     "Alternative of largest polygon voltage";
-  parameter Connection connection=Connection.Series
-    "Specify connection of subsystems in case of mSystems>1";
+  parameter Integer ParDesired(final min=1, final max=mSystems)=mSystems "Desired parallel subsystems";
+  parameter Integer Par=if isPowerOf2(ParDesired) then ParDesired else mSystems "Parallel connected subsystems"
+    annotation(Dialog(enable=false));
+  parameter Integer Ser=integer(mSystems/Par) "Series connected subsystems"
+    annotation(Dialog(enable=false));
```

Comment:
What is the purpose of disabled parameters?

Reply:
Calculating a parameter which shouldn't be changed by the user, but the user can inspect the result. Anyhow, Par and Ser are parameters evaluated before the simulation starts.

Reply:
Parameters should be final then without the Dialog annotation.

Reply:
But then the user has to simulate to inspect these parameters. Anyhow, I followed your advice.